uint8 HeartbeatFlag
uint8 heartbeatrecv_flag
uint32 hrt_recv_num
uint32 hrt_send_num

